lib/openQAcoretest.pm

```diff
@@ -31,6 +31,7 @@ sub upload_openqa_logs {
     chomp(my @logs = qx{find /var/lib/openqa/pool/1/autoinst-log.txt /var/lib/openqa/testresults/*/*/autoinst-log.txt});
     @logs and get_log "(cat @logs ||:)" => 'autoinst-log.txt';
     get_log '(find /var/lib/openqa/pool/ /var/lib/openqa/testresults/ ||:)' => 'find.txt';
+    get_log q|sudo -u geekotest /usr/share/openqa/script/openqa eval -V 'my $jobs = app->minion->jobs; my @r; while (my $j = $jobs->next) { push @r, $j->{result} }; \@r'| => 'openqa_minion_results.txt';
```

Comment:
The file should show up in the assets now, right? Seems like it doesn't? Looking at https://openqa.opensuse.org/tests/4376707#downloads for instance

Reply:
Only
1. in case of a failure when the post_fail_hook is run
2. in multimachine tests where we always upload the logs

Actually I think it could be useful to upload the logs in every case and not just failures.
Can be useful when comparing passed and failed jobs

Reply:
Exactly, and thanks to case 2 we can verify that it actually works: https://openqa.opensuse.org/tests/4376711/logfile?filename=test_running-openqa_minion_results.txt